.springBeans

```diff
@@ -1,15 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Hi @eugenp , unfortunately two pull requests got merged together

My first pull request is only to remove html in templates and second one is to refactor code as per http://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#using-boot-locating-the-main-class spring-boot recommendation.